command/pki_reissue_intermediate.go

```diff
@@ -93,31 +92,25 @@ func (c *PKIReIssueCACommand) Run(args []string) int {
 	}
 
 	parentIssuer := sanitizePath(args[0]) // /pki/issuer/default
+	templateIssuer := sanitizePath(args[1])
 	intermediateMount := sanitizePath(args[2])
 
-	templateCertificateResp, err := client.Logical().Read(sanitizePath(args[1]))
+	issuerBundle, err := readIssuer(client, templateIssuer)
```

Comment:
could we rename this templateIssuerBundle?  I'm worried that this is quite a bit confusing with the parent issuer also being one of the arguments.